packages/mask/public/patches.js

```diff
@@ -83,7 +83,7 @@ globalThis.regeneratorRuntime = undefined
 }
 
 {
-    if (typeof trustedTypes) {
+    if (typeof trustedTypes === 'object') {
```

Comment:
Does it need to check null?

Reply:
it won't be null (it's a web API, which can be only undefined (unsupported) or object)